### PR TITLE
Added information about Haddock and LH with VSCode extension

### PR DESCRIPTION
**Pull request by alcides**
* Added settings for VSCode to disable haddock.
* Added information in the readme about this need, and instructions to replicate in other repos.

Impact:
* Closes Issue #1

### CODE REVIEW

**Review by ranjitjhala — APPROVED** [review]

**Comment by alcides** [comment]
I've merged with main. Is there anything blocking this from being merged?

**Comment by ranjitjhala** [comment]
Nope let’s merge!

**Comment by ranjitjhala** [comment]
I’ll merge as soon as this finishes! LMK if you want merge access - and thanks a million !!

**Pull request merged**
